capa-controller-role/cloud-formation-template.yaml

```diff
@@ -21,6 +21,11 @@ Resources:
             Principal:
               AWS: !Sub "arn:aws:iam::${ManagementClusterAccountID}:user/${InstallationName}-capa-controller"
             Action: "sts:AssumeRole"
+      Tags:
+        - Key: "giantswarm.io/cluster"
```

Comment:
Technically speaking, this role is potentially used to create many different clusters, but this tag makes it sound like this is only used by this cluster.

Reply:
right, makes sense. I guess the `installation` tag should be enough then

Reply:
fixed